### PR TITLE
Admixer: Fix for bid floor issue#1787

### PR DESCRIPTION
**Pull request by avolokha**
- use admixerExt.CustomBidFloor only if imp->bidfloor was zero
- added testcase 
- fixed ZoneId pattern

### CODE REVIEW

**Review by AlexBVolcy — DISMISSED** [review]
LGTM

File: adapters/admixer/admixertest/exemplary/optional-params.json [hunk 1]
```diff
@@ -44,6 +44,29 @@
             }
           }
         }
```
Comment (mansinahar) [inline adapters/admixer/admixertest/exemplary/optional-params.json:46]: Can you please also add a test where both the imp `BidFloor` and `CustomBidFloor` are set to non-zero values and the request to your service has it set to the imp's `BidFloor` value? You can just do this by adding `imp.bidfloor` in one of the imps above and modifying the expected HTTP request accordingly as well

File: static/bidder-params/admixer.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "zone": {
       "type": "string",
       "description": "Zone ID.",
-      "pattern": "^([a-fA-F\\d\\-]{36})$"
+      "pattern": "^([a-fA-F\\d\\-]{32,36})$"
```
Comment (mansinahar) [inline static/bidder-params/admixer.json:11]: Can you please add tests for this change as well in `adapters/admixer/params_test.go`?

**Comment by mansinahar** [comment]
@avolokha Just a friendly reminder to see if you are able to add the above suggested tests

**Review by SyntaxNode — CHANGES_REQUESTED** [review]
Please respond to the PR feedback.

File: static/bidder-params/admixer.json (hunk 2 above)
Comment (Denis-Admixer) [reply static/bidder-params/admixer.json:11]: Hi @mansinahar , sure, we will add tests.

**Comment by avolokha** [comment]
Hi Team,
I am sorry for a big delay here. I've added tests for non zero values(imp->Bidfloor and customFloor) and our "Zone" parameter validation.

**Review by mansinahar — APPROVED** [review]

**Review by AlexBVolcy — APPROVED** [review]
Thank you for responding to feedback, this looks solid to me!

**Review by SyntaxNode — APPROVED** [review]

**Pull request merged**